packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/index.tsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+import { bindActionCreators, Dispatch } from 'redux'
+
+import { actions } from 'data'
 
 import DebitCard from './template'
```

Comment:
Could you use absolute imports?

Reply:
Could you give an example of what you mean? I just kept the structure of another component of the wallet